functions/get_puppet_servers.pp

```diff
@@ -0,0 +1,11 @@
+function rsan::get_puppet_servers() {
+  $puppet_servers =
```

Comment:
there is a syntax issue here, best PDK validate

Reply:
Syntax issue on this should be solved in my next commit from my local repo :)